covid19zonecare/src/main/java/com/cognizant/callforcode/zonecare/entity/Pincode.java

```diff
@@ -6,9 +6,10 @@
 @Entity
 public class Pincode {
 	
-	@Id
+	
 	private int Id;
 	private String pincode_ward;
+	@Id
```

Comment:
is Pincode the primary key here? @DeepaBaddur

Reply:
before id was primary it is autogenerated, which we can't track. So i made it pincode is primary key.

Reply:
Pincode can be repeated here. Please check. There will be multiple wards for one pincode. Please make pincode_ward primary key or keep id as it is.